src/app/[locale]/(auth)/forgot-password/page.tsx

```
@@ -1,6 +1,6 @@
 import React from "react";
 import ForgotPasswordLayout from "./_components/layout/forgot-passowrd-layout";
-import Link from "next/link";
+import { Link } from "@i18n/navigation";
```

ESLint

[error] 'Link' is defined but never used.

(@typescript-eslint/no-unused-vars)